kinoml/workflows/chembl22_cnn.py

```diff
@@ -0,0 +1,188 @@
+'''
+This script uses the pre-processed data file df_reduced.pkl that is generated from the ChEMBL22 data set.
+
+Type of target used: single-target (i.e. ligand-based model)
+Type of featurizer for the ligand: One Hot Encoding of SMILES
+Type of model: a Convolutional Neural Network (CNN) using Pytorch
+
+Please run in terminal the following command:
+$ export PYTHONPATH=$(dirname $(pwd))
```

Comment:
We don't want to mess up with `PYTHONPATH` (or `sys.path` runtime modifications). Development should happen with the package installed in development mode:

```
pip install -e .
```